README.md

```diff
@@ -10,3 +10,16 @@ Still in development stage, waiting for your contribution!
 * no overhead for GPU-CPU memory copy
 * can run in parallel on multiple GPUs
 
+### This Fork:
```

Comment:
Could you move fragments of this paragraph from README to appropriate commit messages?